src/pages/resolve/components/NomineesTable.vue

```diff
@@ -0,0 +1,259 @@
+      } catch (err) {
+        console.log("removeSelf error: ", err);
+      }
+    },
+    async enterElection() {
+      const enterElectionAction = [
+        {
+          account: "testtelosarb",
+          name: "candaddlead",
+          data: {
+            nominee: this.$store.getters["accounts/account"],
+          },
+        },
+      ];
+      try {
+        await this.$store.$api.signTransaction(enterElectionAction);
+      } catch (err) {
+        console.log("removeSelf error: ", err);
+      }
+    },
+    closeModal() {
+      this.nominate = false;
+    },
+  },
+  computed: {
+    isNominateButtonVisible() {
+      if (!this.elections || !this.configData) return;
+      const isAuthenticated = this.$store.getters["accounts/isAuthenticated"];
+      const account = this.$store.getters["accounts/account"];
+      const isAlreadyNominated = this.nomineeData.find(
+        (nominee) => nominee.nominee_name === account
+      );
+      if (isAuthenticated && !isAlreadyNominated) {
+        return true;
+      }
+      return false;
+    },
+    isEnterElectionButtonVisible() {
+      if (!this.elections || !this.configData) return;
+      const account = this.$store.getters["accounts/account"];
+      const { current_election_id } = this.configData;
+      const currentElection = this.elections.find(
+        (election) => election.election_id === current_election_id
+      );
+      const isAlreadyCandidate = !!currentElection.candidates.find(
+        (candidateData) => candidateData.name === account
+      );
+      const endAddCand = new Date(
+        currentElection.end_add_candidates_ts
+      ).getTime();
+      const isPastAddCandidates = Date.now() > endAddCand;
+
+      const isElectionCreated = currentElection.status === 1;
+      if (isElectionCreated && !isAlreadyCandidate && !isPastAddCandidates) {
+        return true;
+      }
+      return false;
+    },
+    isRemoveCandidateButtonVisible() {
+      if (!this.elections || !this.configData) return;
+      const account = this.$store.getters["accounts/account"];
+      const { current_election_id } = this.configData;
+      const currentElection = this.elections[current_election_id];
+      const isAlreadyCandidate = currentElection.candidates.find(
+        (candidateData) => candidateData.name === account
+      );
+      const endAddCand = new Date(
+        currentElection.end_add_candidates_ts
+      ).getTime();
+      const isPastAddCandidates = Date.now() > endAddCand;
+
+      const isElectionCreated = currentElection.status === 1;
+      if (isElectionCreated && isAlreadyCandidate && !isPastAddCandidates) {
+        return true;
+      }
+      return false;
+    },
+    nomineeData() {
+      return this.$store.state.resolve.nominees || [];
+    },
+    configData() {
+      return this.$store.state.resolve.config || [];
+    },
+    elections() {
+      return this.$store.state.resolve.elections || [];
+    },
+  },
+};
+</script>
+
+<style scoped>
```

Comment:
Verify your styles don't break anything outside this scope.

```suggestion
<style>
```